change/@internal-react-components-168885cc-5d87-4420-badb-9b6b242d6ac0.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "created button component",
```

Comment:
The changelog comment should make sense on its own in a CHANGELOG file.

How about "internal: Introduce a component to cycle through available cameras"

Reply:
OR as per James' created a guideline for such comments long ago.
"Add component for cycling through available cameras"

:)